lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -16,7 +16,8 @@ import {
     ProtocolMode,
     StaticAuthorityOptions,
     AuthenticationResult,
-    UrlString,
+    createClientConfigurationError,
```

Comment:
nit: should this be capitalized?

Reply:
This is inline with all other error naming, and this is probably a `function` exported.